subworkflows/local/short_variant_calling/main.nf

```diff
@@ -18,61 +16,44 @@ workflow SHORT_VARIANT_CALLING {
     ch_bed         // channel: [optional] [ val(meta), path(input_bed) ]
 
     main:
-    ch_snp_calls_vcf  = Channel.empty()
-    ch_snp_calls_gvcf = Channel.empty()
-    ch_combined_bcf   = Channel.empty()
-    ch_versions       = Channel.empty()
+    ch_versions = Channel.empty()
 
     DEEPVARIANT ( ch_bam_bai_bed, ch_fasta, ch_fai, [[],[]] )
     ch_versions = ch_versions.mix(DEEPVARIANT.out.versions)
 
-    // Collect VCFs
-    ch_snp_calls_vcf  = ch_snp_calls_vcf.mix(DEEPVARIANT.out.vcf)
-
-    // Collect GVCFs
-    ch_snp_calls_gvcf = ch_snp_calls_gvcf.mix(DEEPVARIANT.out.gvcf)
-
-    // DV gVCFs
-    TABIX_DV(ch_snp_calls_gvcf)
-    ch_versions = ch_versions.mix(TABIX_DV.out.versions)
-
-    ch_snp_calls_gvcf
-        .groupTuple() // size not working here if there are less than specifed regions..
-        .join(TABIX_DV.out.tbi.groupTuple())
-        .set{ bcftools_concat_dv_in }
+    // gVCF
+    DEEPVARIANT.out.gvcf
+        .groupTuple()
```

Comment:
Is it possible to add `size` to the groupTuple operator a groupKey here?

Reply:
Let's do a groupKey (#202), I will fix it in a separate PR.

Reply:
Actually let's do it now instead.